stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/extensions/internal/TextView.kt

```diff
@@ -39,36 +38,6 @@ internal fun TextView.setStartDrawable(@DrawableRes id: Int) {
     setStartDrawable(ContextCompat.getDrawable(context, id))
 }
 
```

Comment:
Unused code and, beside that, looks like the Drawable is tinted without mutating.